src/conversations/Conversations.ts

```diff
@@ -454,7 +447,9 @@ export default class Conversations {
     const context = {
       conversationId: `xmtp.org/groups/${groupID}`,
       metadata: {
-        initialMembers: initialMembers.join(','),
+        initialMembers: [this.client.address, ...new Set(initialMembers)].join(
```

Comment:
```suggestion
        initialMembers: [..new Set(initialMembers.concat(this.client.address))].join(
```

That would make sure that if the sender is in the list of initial members they don't get duped.